tests/test_crosstalk.py

```diff
@@ -0,0 +1,66 @@
+# This file is part of ci_cpp.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import unittest
+import lsst.utils.tests
+
+
+class CrosstalkTestCases(lsst.utils.tests.TestCase):
+
+    def setup_independentFrame(self):
+        """Missing data.
```

Comment:
Do you mean that this test will be filled in later but the data are missing? Does it really need to be here at all? Will you forget?

Reply:
Added reference to DM-26396, which will prevent forgetting.